app/logic/courseManagement.py

```diff
@@ -0,0 +1,56 @@
+from app.models.courseInstructor import CourseInstructor
+from app.models.courseStatus import CourseStatus
+from app.models.course import Course
+from app.models.term import Term
+
+
+def pendingCourses(termId):
+    '''
+    Queries the database to get all the neccessary information for
+    non approved and non completed courses.
+    '''
+

```

Comment:
You should be able to get course and instructor info in one query. An alternate option would be to use the courseInstructors backreference that was added to Course